packages/tldraw/src/lib/ui/components/primitives/TldrawUiIcon.tsx

```diff
@@ -41,27 +49,48 @@ export const TldrawUiIcon = memo(function TldrawUiIcon({
 	}, [ref, asset, icon])
 
 	if (icon === 'none') {
-		return (
-			<div
-				className={classNames(
-					'tlui-icon tlui-icon__placeholder',
-					{ 'tlui-icon__small': small },
-					className
-				)}
-			/>
-		)
+		return <div className={classNames('tlui-icon', { 'tlui-icon__small': small }, className)} />
 	}
 
 	return (
 		<div
 			{...props}
 			ref={ref}
-			className={classNames('tlui-icon', { 'tlui-icon__small': small }, className)}
+			className={classNames('tlui-icon tlui-icon__asset', { 'tlui-icon__small': small }, className)}
```

Comment:
ever so slight nit here (and below):
```suggestion
			className={classNames('tlui-icon', 'tlui-icon__asset', { 'tlui-icon__small': small }, className)}
```
just for readability and for prettier to do it's thing